SPDX-FileCopyrightText: 2021 Melg Eight <public.melg8@gmail.com>

SPDX-License-Identifier: GPL-3.0-or-later

Remove dependency on propername.h.
We do that, because inclusion of propername.h from gnulib brings dependencies
on gperf generated files. We don't have gperf at this stage so we can't
regenerate them.

--- ./src/system.h
+++ ./src/system.h
@@ -361,13 +361,13 @@ enum
 #include "version-etc.h"
 #undef emit_bug_reporting_address
 
-#include "propername.h"
 /* Define away proper_name (leaving proper_name_utf8, which affects far
    fewer programs), since it's not worth the cost of adding ~17KB to
    the x86_64 text size of every single program.  This avoids a 40%
    (almost ~2MB) increase in the on-disk space utilization for the set
    of the 100 binaries. */
 #define proper_name(x) (x)
+#define proper_name_utf8(x, y) (x, y)
 
 #include "progname.h"
 
